src/client/pages/OfferDebugger/components/QuoteData.tsx

```diff
@@ -132,7 +132,19 @@ const getCurrentAvailableQuoteData = (
   const currentQuoteTypeData = marketQuoteTypes.find(
     ({ value }) => value === currentQuoteType,
   )
-  return currentQuoteTypeData
+
+  const randomId = Math.random()
+    .toString(36)
+    .substr(2, 5)
+  const email = `sven.svensson.${randomId}@hedvig.com`
```

Comment:
👍 Just so I understand, in what way did the same email conflict? :)

Reply:
ahhh, that's actually how generic auth is affecting us --- now the email needs to be unique for each user. You are not able to create a quote with an email that has been used to sign a quote already.